Sources/BraintreePayPalMessaging/BTPayPalMessagingClient.swift

```diff
@@ -0,0 +1,150 @@
+import UIKit
+import SwiftUI
+import PayPalMessages
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Use `BTPayPalMessagingClient` to display PayPal messages to promote offers such as Pay Later and PayPal Credit to customers.
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public class BTPayPalMessagingClient: UIView {
+
+    // MARK: - Properties
+
+    public weak var delegate: BTPayPalMessagingDelegate?
+
+    var apiClient: BTAPIClient
+
+    // MARK: - Initializers
+
+    ///  Initializes a PayPal Messaging client.
+    /// - Parameter apiClient: The Braintree API client
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+
+        super.init(frame: .zero)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public Method
+
+    /// Creates a `BTPayPalMessagingClient` to be displayed to promote offers such as Pay Later and PayPal Credit to customers.
+    /// - Parameter request: an optional `BTPayPalMessagingRequest`
```

Comment:
This is confusing to me since this method doesn't return anything.

Reply:
Updated: https://github.com/braintree/braintree_ios/pull/1153/commits/a9c78b44c62adce131a7a4b450be5c201007b1cd